exporter/collector/README.md

```diff
@@ -317,6 +318,13 @@ additional flexibility in parsing log severity from incoming entries.
 
 By default, the exporter sends telemetry to the project specified by `project` in the configuration. This can be overridden on a per-metrics basis using the `gcp.project.id` resource attribute. For example, if a metric has a label `project`, you could use the `groupbyattrs` processor to promote it to a resource label, and the `resource` processor to rename the attribute from `project` to `gcp.project.id`.
 
+### Multi-Project quota usage
+
+The `gcp.project.id` label can be combined with the `destination_project_quota` option to attribute quota usage to the project parsed by the label. This feature is currently only available
+for traces and metrics. The Collector's default service account will need `roles/serviceusage.serviceUsageConsumer` IAM permissions in the destination quota project.
+
+Note that this option will not work  if a quota project is already defined in your Collector's GCP credentials. In this case, the telemetry will fail to export with a "project not found" error.
```

Comment:
It might be nice to provide instructions on how to ensure the quota project isn't set on the credentials.  `gcloud auth application-default set-quota-project ""` doesn't seem to work.

Reply:
I just manually removed the line from my `.config/gcloud/application_default_credentials.json` file. Not sure if there is an official/recommended way to do this but I can at least add that step